python/BUILD

```diff
@@ -34,6 +34,7 @@ pytest_test(
     name = "hello_pytest_test",
     srcs = ["hello_pytest_test.py"],
     shard_count = 2,
+    tags = ["todo-pe1188-remote-incompatible"],
```

Comment:
Is there some built-in automation to exclude tests with this sort of tag? Or is there a query in the CI command for python tests that excludes this? Just curious how it works.